### PR TITLE
2023-01 Maintenance

### PR DESCRIPTION
**Pull request by dtbaker**
## Changes

- Updated npm packages to latest
- Removed deprecated packages / tooling
- Fixed a bug with item ratings
- Updated our URL to https://www.envato.com/lp/market-plugin/
- Included a fix from the community: https://github.com/envato/wp-envato-market/pull/228

### CODE REVIEW

File: inc/admin/functions.php [hunk 1]
```diff
@@ -188,12 +188,14 @@ function envato_market_themes_column( $group = 'install' ) {
...
+							if ( is_array( $theme['rating'] ) ) {
+							    $count = ! empty( $theme['rating']['count'] ) ? $theme['rating']['count'] : 0;
+							    $rating = ! empty( $theme['rating']['rating'] ) ? (int) $theme['rating']['rating'] : 0;
 								wp_star_rating(
 									array(
-										'rating' => $theme['rating']['count'] > 0 ? ( $theme['rating']['rating'] / 5 * 100 ) : 0,
+										'rating' => $count > 0 ? ( $rating / 5 * 100 ) : 0,
 										'type'   => 'percent',
-										'number' => $theme['rating']['count'],
+										'number' => $count,
```
Comment (blimpage) [inline inc/admin/functions.php:198]: Should this bug fix be mentioned in the changelogs too?

Comment (dtbaker) [reply inc/admin/functions.php:198]: Yes! :loading:

File: languages/envato-market.pot [hunk 2]
```diff
@@ -1,34 +1,40 @@
...
+#: docs/dist/envato-market/inc/admin/class-envato-market-admin.php:78
+#: docs/dist/envato-market/inc/admin/class-envato-market-admin.php:88
+#: docs/dist/envato-market/inc/class-envato-market-api.php:80
+#: docs/dist/envato-market/inc/class-envato-market-api.php:90
+#: docs/dist/envato-market/inc/class-envato-market-github.php:87
+#: docs/dist/envato-market/inc/class-envato-market-github.php:97
+#: docs/dist/envato-market/inc/class-envato-market-items.php:100
+#: docs/dist/envato-market/inc/class-envato-market-items.php:110
+#: docs/dist/envato-market/inc/class-envato-market.php:149
+#: docs/dist/envato-market/inc/class-envato-market.php:159
```
Comment (blimpage) [inline languages/envato-market.pot:37]: There are lots of new inclusions from `docs/dist/` in here, are they meant to be included in this file?

Comment (blimpage) [reply languages/envato-market.pot]: Ah never mind, you've already removed them 😄

**Review by blimpage — APPROVED** [review]
I wasn't able to thoroughly review the ESLint opinion changes, but the rest looks good to me! Good call on removing some broken stuff.

File: languages/envato-market.pot (hunk 2 above)
Comment (dtbaker) [reply languages/envato-market.pot]: Yes this was a mistake, I removed those `exclude` options trying to get the updated `makepot` tool working. Forgot to bring them back.

They're back now and those entries shouldn't make it into the potfile any more: https://github.com/envato/wp-envato-market/pull/282/commits/5a60d80be9596a3a9f516d105689865b9a6f8585

**Pull request merged**